src/main/java/com/google/sps/servlets/EnterQueue.java

```diff
@@ -159,24 +174,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             }
           }
         }
-        response.sendRedirect(STUDENT_QUEUE + classCode);
-      } else {
-        Key classKey = KeyFactory.stringToKey(classCode);
-        Entity classEntity = datastore.get(classKey);
+        response.setStatus(201);
```

Comment:
Is `201` the correct code here. A `201` status represents a resource that was created.

Also use the constants in `HttpServletResponse` for status codes.

Reply:
Could you use a constant here as well?